poc/intel/manifests/notebooks.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: notebooks-home
+  namespace: dss
+  labels:
+    app: dss-notebooks
+    app.kubernetes.io/name: dss-notebooks
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
```

Comment:
I think we should add 
```
storageClassName: microk8s-hostpath
volumeMode: Filesystem
```

Its used by default in this microk8s setup but for sake of clarity I would add it.

Reply:
done